tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -245,13 +243,11 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
             result["hashType"] = file_hash_type
             copied_file_urls.append(file_source_uri)
             LOGGER.info(
-                "Copied {file_filepath} into glacier storage bucket {default_bucket}.",
-                file_filepath=file_filepath,
-                default_bucket=destination_bucket,
+                f"Copied {file_filepath} into glacier storage bucket {destination_bucket}."
             )
             # Add file record to metadata SQS message
             LOGGER.debug(
-                "Adding the files dictionary to the SQS body {result}.", result=result
+                "Adding the files dictionary to the SQS body. {result}.", result=result
```

Comment:
Should this be changed to `f` string like the others you fixed?

Reply:
Nope. Result can contain `{}` and Cumulus logger treats that as requiring a parameter for replacement.

Reply:
ahh, ok.